src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,415 @@
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
+
+* Deploy the clusters with `subctl join --natt=false`
+
+{{% notice note %}}
+The first alternative is the preferred one, since it will allow the eventual
+addition of a cluster on a public cloud, or a remote site with no direct IP
+reachability or NATed reachability.
+{{% /notice %}}
+
+### All public cloud, with some private reachability
+
+In this case case, A & B cluster gateways have direct reachability over their private
+IPs (10.0.0.1 and 10.1.0.1) possibly with large MTU capabilities, happening the
+same for C & D cluster gateways (192.168.0.4 and 192.168.128.4).
+
+Between A & C , A & D, B & C, and B & D reachability is still possible over
+the public IP, the IP packets will undergo DNAT + SNAT translation at the
+border via the elastic or floating IP, and also while on transit via the public
+network the MTU will be limited to 1500 bytes or less.
+
+<!-- source: https://app.diagrams.net/#G1IORJ9qW95qJ15P6Xk6idEfI4jj9Uo84l -->
+![Public vs on-premises](/images/natt/nat_scenario_all_public.svg)
+
+#### Endpoints
+
+| Endpoint | Private IP      | Public IP |
+|:---------|:----------------|:----------|
+|   A      | 10.0.0.1        | 1.1.1.1   |
+|   B      | 10.1.0.1        | 1.1.1.2   |
+|   C      | 192.168.0.4     | 2.1.1.1   |
+|   D      | 192.168.128.4   | 2.1.1.2   |
+
+#### Connections
+
+| Left Cluster |  Left IP     | Left Port | Right Cluster  | Right IP       | Right Port | NAT   |
+|:-------------|:-------------|:----------|:---------------|:---------------|:-----------|:------|
+| A            |  10.0.0.1    | 4500      | B              | 10.1.0.1       | 4500       | no    |
+| C            |  192.168.0.4 | 4500      | D              | 192.168.128.4  | 4500       | no    |
+| A            |  1.1.1.1     | 4500      | C              | 2.1.1.1        | 4500       | yes   |
+| A            |  1.1.1.1     | 4500      | D              | 2.1.1.2        | 4500       | yes   |
+| B            |  1.1.1.2     | 4500      | C              | 2.1.1.1        | 4500       | yes   |
+| B            |  1.1.1.2     | 4500      | D              | 2.1.1.2        | 4500       | yes   |
+
+The recommended configuration is enabling the NATT discovery protocol on
+all ports, and make sure that the gateways have such port open (as well as the
+encapsulation port `4500/udp` by default)
+
+```bash
+kubectl label node $GWA --kubeconfig A gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWB --kubeconfig B gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWC --kubeconfig C gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWD --kubeconfig D gateway.submariner.io/natt-discovery-port=4490
+
+# restart the gateways to pick up the new setting
+for cluster in A B C D;
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
```

Comment:
Remove since NATT discovery protocol is enabled by default?

Reply:
Makes sense, will do on separate PR.

Reply:
Done now